nautobot/docs/user-guide/administration/configuration/optional-settings.md

```diff
@@ -114,6 +115,7 @@ These environment variables may be used to specify the values:
 * `NAUTOBOT_BRANDING_FILEPATHS_ICON_MASK`
 
 +++ 2.1.0
+    * `NAUTOBOT_BRANDING_FILEPATHS_HEADER_BULLET`
     * `NAUTOBOT_BRANDING_FILEPATHS_NAV_BULLET`
```

Comment:
Markdownlint seems unhappy with this though the rendered page looks fine to me. May be simplest just to disable rule `MD037/no-space-in-emphasis` inline?